### ISSUE TITLE
Can you tell me what accuracy have you achieved?

### ISSUE THREAD

**Issue by greathope** [Cadene/vqa.pytorch#26]
with this command on vqa v1.0:
python train.py --vqa_trainsplit trainval --path_opt options/vqa/mutan_att_trainval.yaml

**Comment by Cadene** [comment]
@greathope As far as I remember, we got those results on the testset (with our old Torch7 code, but you should get the same results with this code in pytorch):

```
Overall Accuracy is: 64.77
Per Answer Type Accuracy is the following:
other : 54.47
number : 38.43
yes/no : 83.81
```

**Comment by greathope** [comment]
@Cadene with trainval datasets? Actually I got 65.42...

**Comment by Cadene** [comment]
That's good news, isn't it?

It might be related to this little change in architecture: https://github.com/Cadene/vqa.pytorch/issues/18

What do you think? @greathope

**Comment by eustcPL** [comment]
@Cadene how about: vqav1.0 python train.py --vqa_trainsplit trainval --path_opt options/vqa/mlb_att_trainval.yaml, what accuracy did you acheived?
